Source/Schema.NET/OneOrMany{T}.cs

```diff
@@ -125,7 +125,7 @@ public int Count
         /// <param name="list">The list of values.</param>
         /// <returns>The result of the conversion.</returns>
 #pragma warning disable CA2225 // Operator overloads have named alternates
-        public static implicit operator OneOrMany<T>(List<T> list) => new OneOrMany<T>(list);
+        public static implicit operator OneOrMany<T>(List<T> list) => new OneOrMany<T>(list.Where(x => x != null && !(x.GetType() == typeof(string) && string.IsNullOrWhiteSpace(x as string))));

```

Comment:
Same here.

Also, this code was relying on the constructor to throw if the list was null but we can't do that anymore, so you'll need to check for null and throw ArgumentNullException.

Reply:
Doesn't look like you can raise an exception within an implicit operator method. Using null propagation works though (via `list?.Where(...)`), and probably cleaner

Reply:
Cool.